solutions/security/get-started/get-started-detect-with-siem.md

```diff
@@ -0,0 +1,181 @@
+
+:::{dropdown} Steps to modify an endpoint policy
+1. From the left navigation menu, go to **Assets** → **Endpoints** → **Policies**. 
+2. From the list, select the policy you want to configure. The policy configuration page appears.
+3. On the **Policy settings** tab, for each protection, switch the protection level from `Prevent` to `Detect`. 
+4. Review and configure the event collection and antivirus settings as appropriate. 
+5. Once you're finished making changes, click **Save** in the lower-right corner to update the policy.  
+
+:::{note}
+For a comprehensive explanation of all endpoint protections and policy settings, refer to [Configure an integration policy](/solutions/security/configure-elastic-defend/configure-an-integration-policy-for-elastic-defend.md). 
+:::
+:::
+:::::
+
+## Add Elastic prebuilt detection rules
+
+Detection rules allow you to monitor your environment by searching for source events, matches, sequences, or {{ml}} job anomaly results that meet their criteria. When a rule’s criteria are met, {{elastic-sec}} generates an alert. While you can create your own rules tailored for your environment, Elastic ships out-of-the-box prebuilt rules that you can install. Remember that if you installed {{elastic-defend}}, the Endpoint Security rule is already enabled. 
+
+:::{dropdown} Steps to install and enable prebuilt rules
+1. On the **Get Started** page, scroll down to the **Configure rules and alerts** section. 
+2. Click **Install Elastic rules**, then **Add Elastic rules**. The Rules page displays. 
+3. At the top of the page, click **Add Elastic rules**. The badge next to it shows the number of prebuilt rules available for installation.  
+4. Use the search bar and **Tags** filter to find the rules you want to install. For example, to filter by operating system, search for the appropriate OS (such as `macOS`) from the **Tags** menu. 
+5. Once you've filtered the rules, confirm that the rules displayed are the ones you'd like to install. If you'd like to learn more about any rule before installing it, click on the rule name to expand the rule details flyout. Here's an example of one: 
+
+    :::{image} /solutions/images/security-gs-siem-rule-details.png 
+    :alt: Rule details flyout
+    :screenshot:
+    :::
+
+6. Select the check box next to the rules you want to install. To select all rules on the page, select the check box to the left of the **Rule** column heading. We recommend installing all the rules for your operating system, but you can install whichever rules you're comfortable with to start. You can always install more later.  
+7. Click ![Vertical boxes button](/solutions/images/serverless-boxesVertical.svg "") → **Install and enable** to install and start running the rules. Once you enable a rule, it starts running on its configured schedule.
+
+    :::{note}
+    You must install _and_ enable the rule for it to start running; therefore, do not select the **Install _x_ rules** option.  
+    :::
```

Comment:
I think "do not select the Install x rules option" is a bit too strong statement, I suggest saying something "to speed up the process for the quick start, select Install and Enable option. The alternative is to enable a rule once it is installed from Installed rules table.